src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+$--day-block-width: 100px;
+$--day-block-height: 100px;
+$days: ("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6);
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+
+  max-width: #{$--day-block-height * 7 + 20};
+  padding: 30px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $--day-block-width;
+    height: $--day-block-width;
+    box-sizing: border-box;
+    border: 1px solid black;
+
+    background-color: #eee;
+
+    transition: all 0.5s;
```

Comment:
list styles to which you apply transition instead of "all", 8'th point in checklist

Reply:
✅Done